albatross.ml

```diff
@@ -38,6 +48,35 @@ struct
     in
     Ok (Vmm_trie.fold path t.policies (fun name p acc -> (name, p) :: acc) [])
 
+  let policy_resource_used () t =
+    let root_policy =
+      match policy t with
+      | Ok p -> ( match p with Some p -> p | None -> empty_policy)
+      | Error _ -> empty_policy
```

Comment:
should we fail the request at this point?

Reply:
I guess so, as mentioned somewhere else, I'd expect this endpoint to fail if there's no albatross connected that gave us a root policy. Since we don't store the policy persistently in mollymawk, we need a working albatross which stores the policy.